### ISSUE TITLE
build error with node v11

### ISSUE THREAD

**Issue by pkulijing** [nolanlawson/local-npm#182]
## Detailed Description

Trying to install with `yarn global add local-npm`, but got the following build error:

	In file included from ../../nan/nan.h:192:0,
	                 from ../src/batch.cc:3:
	../../nan/nan_maybe_43_inl.h: In function ‘Nan::Maybe<bool> Nan::ForceSet(v8::Local<v8::Object>, v8::Local<v8::Value>, v8::Local<v8::Value>, v8::PropertyAttribute)’:
	../../nan/nan_maybe_43_inl.h:112:15: error: ‘class v8::Object’ has no member named ‘ForceSet’
	   return obj->ForceSet(isolate->GetCurrentContext(), key, value, attribs);

Full build output is attached.
[log.txt](https://github.com/local-npm/local-npm/files/3171093/log.txt)

## Your Environment

* Operating System and version: Ubuntu 16.04.1
* node: v11.15.0
* yarn: 1.15.2
* npm: 6.7.0

**Comment by pkulijing** [comment]
Tested using node[ dockers of different versions](https://hub.docker.com/_/node). The installation works on node:8, but fails for node:10 and node:11.

**Comment by lordadamson** [comment]
same issue here with node 12. Will try node 8. Thanks for reporting.

**Comment by lordadamson** [comment]
node 8 works. Thanks a lot.

**Comment by ryanmjacobs** [comment]
Yeah, I just pulled my hair out for half an hour... Docker's `node:8` worked for me.

Should this be mentioned in the main readme? Took me a little while to find this information.

**Comment by Moanrisy** [comment]
I'm installing with 8.17.0 work too
